wpiutil/src/main/native/cpp/MimeTypes.cpp

```diff
@@ -0,0 +1,69 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "wpi/MimeTypes.h"
+
+#include "wpi/StringMap.h"
+
+using namespace wpi;
+
+// derived partially from
+// https://github.com/DEGoodmanWilson/libmime/blob/stable/0.1.2/mime/mime.cpp
+StringRef MimeTypeFromPath(StringRef path) {
```

Comment:
A free function won't put itself in the namespace.  You need to either make this `wpi::MimeTypeFromPath` or wrap the file in `namespace wpi {}` (in which case you can get rid of the using directive).